content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -350,8 +350,6 @@ The tracking issue for this problem is [here](https://github.com/kubernetes/kube
 
 ## The NodeRegistration.Taints field is omitted when marshalling kubeadm configuration
 
-*Note: This [issue](https://github.com/kubernetes/kubeadm/issues/1358) only applies to tools that marshal kubeadm types (e.g. to a YAML configuration file). It will be fixed in kubeadm API v1beta2.*
```

Comment:
removing this note since it affects the v1beta1 API which is being removed in 1.22.